Dockerfile

```diff
@@ -1,3 +1,2 @@
-FROM alpine:latest
-
-CMD ["whoami"]
+FROM maven:3.6.3-jdk-8-slim
+ENTRYPOINT [ "mvn test" ]
```

Comment:
This doesn't do anything?  This Dockerfile needs to install hiverunner and its dependancies, so it can run tests locally.